src/shared/locales/en/translation.json

```diff
@@ -63,6 +63,8 @@
         "password": "Password",
         "unrecognisedPassword": "Unrecognised password",
         "unrecognisedPasswordExplanation": "The password was not recognised. Please try again.",
+        "unrecognisedAccount": "Unrecognised account",
+        "unrecognisedAccountExplanation": "Account named {{accountName}} was not recognised. Consider reset your wallet.",
```

Comment:
Sorry, I forgot one thing in the earlier review. Could you change this to `Consider resetting your wallet.` or `Please go to Advanced Settings reset the wallet.`?

Reply:
Not a problem.

Reply:
Just realized that the Advanced Settings can't be accessed if you can't log in 😅 `Please reset the wallet` might be better here in that case

Reply:
hehe, I should have thought of that as well